docs/policy/container-release.md

```diff
@@ -17,32 +17,43 @@ Similar to our existing RPM infrastructure, container image sources, build logs,
 publicly available repositories (e.g. GitHub, Docker Hub) for collaboration and traceability.
 Additionally, container images distributed by the OSG Software team will be based off of the latest version of a 
 [supported platform](https://opensciencegrid.org/docs/release/supported_platforms/) with software installed from OS,
-EPEL, and [OSG release](../policy/software-release.md#yum-repositories) Yum repositories with select packages
-installed from `osg-development`.
+EPEL, and [OSG](../policy/software-release.md#yum-repositories) Yum repositories.
 
 Tags
 ----
 
-OSG Software container images will be tagged with at least one of the following tags:
+OSG Software container images will be built at least weekly and tagged with one of the following:
 
-| Tag       | Description                                                                                                                                                       |
-|-----------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `fresh`   | Images that build successfully and pass automated tests. Intended for users that need the latest fixes and features.                                              |
-| timestamp | Each `fresh` image  is also tagged with a timestamp reflecting their build date and time. Intended for rollback in case of issues with the current `fresh` image. |
-| `stable`  | `fresh` images that pass acceptance testing; approved by the Release Manager. Intended for stable production use.                                                 |
+| Tag                            | Description                                                                                                              |
+|--------------------------------|--------------------------------------------------------------------------------------------------------------------------|
+| `testing`                      | For the latest fixes and features. Built with RPMs from the `osg-testing` and/or `osg-upcoming-testing` Yum repositories |
```

Comment:
Will we have `3.5-testing` and `3.6-testing`, or just whatever is newest? In either case ,you might want to mention that.

Reply:
I think whatever's newest, which is something carried over from the old policy: https://github.com/opensciencegrid/technology/pull/877/files#diff-0effce1664d7148577b2521054df51009a786873989c5ea9df836c47809df2a5R16-R20.

Reply:
Sounds good; what are we going to do for 'preview versions' (i.e. testing stuff before 3.6 and el8 are ready)?

Reply:
We could just use the `rc.*` tags for that. For 3.6,  we can just use `testing` and `release` out of the gate since I want to start this policy with 3.6.